seatunnel-api/src/main/java/org/apache/seatunnel/api/table/factory/TableSinkFactory.java

```diff
@@ -32,11 +33,28 @@
 
     /**
      * We will never use this method now. So gave a default implement and return null.
+     *
      * @param context TableFactoryContext
      * @return
      */
     default TableSink<IN, StateT, CommitInfoT, AggregatedCommitInfoT> createSink(TableFactoryContext context) {
         throw new UnsupportedOperationException("unsupported now");
     }
 
+    /**
+     * This method is called by SeaTunnel Web to get the full option rule of a sink.
+     * Please don't overwrite this method.
+     * @return
+     */
+    default OptionRule fullOptionRule() {
+        OptionRule optionRule = optionRule();
+        if (optionRule == null) {
+            throw new FactoryException("OptionRule can not be null");
+        }
+
+        OptionRule sinkCommonOptionRule =
+            OptionRule.builder().optional(SinkCommonOptions.SOURCE_TABLE_NAME, SinkCommonOptions.PARALLELISM).build();
+        optionRule.getOptionalOptions().addAll(sinkCommonOptionRule.getOptionalOptions());
+        return optionRule;
+    }
```

Comment:
IMO, Add this code snippet to the Util class

Reply:
> IMO, move this code snippet to the Util class

Ok, I will do it after we removed V1 code from branch dev.

Reply:
done